src/components/NoteLabelItem/NoteLabelItem.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { useNote } from "../../contexts";
+import { bxIcons } from "../../data/icons";
+import "./NoteLabelItem.css";
+
+export const NoteLabelItem = ({ label, closeBtn }) => {
+	const { noteDispatch } = useNote();
+	const [showCloseBtn, setShowCloseBtn] = useState(false);
+
+	const removeLabelBtn = (label) => {
+		noteDispatch({
+			type: "REMOVE_LABEL",
+			payload: label,
+		});
+	};
+
+	return (
+		<li
+			className="label__li"
+			onMouseEnter={() => setShowCloseBtn(true)}
+			onMouseLeave={() => setShowCloseBtn(false)}
+		>
+			<div className="label__item">{label}</div>
+			{closeBtn ? (
+				<button
+					className={`label_btn__cross ${showCloseBtn ? "show_close_btn" : ""}`}
```

Comment:
good way of using template literals like this I didn't know about it, thanks

Reply:
My pleasure, you got to learn.